### PR TITLE
Added 2 more unicode characters

### PR DESCRIPTION
**Pull request by HBIDamian**
Added: 
- Token  U+E105
- Immersive Reader Button  U+E104

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,95 +9,97 @@ Minecraft uses resource packs to assign characters (glyphs) to different unicode
...
+|         Left Grab Button         |     Rift     |          |  U+E0E3 |                      ![](images/rift-lg.png)                     |
+|         Right Grab Button        |     Rift     |          |  U+E0E4 |                      ![](images/rift-rg.png)                     |
+|         Left Stick Button        |     Rift     |          |  U+E0E5 |                     ![](./images/rift-ls.png)                    |
+|        Right Stick Button        |     Rift     |          |  U+E0E6 |                     ![](./images/rift-rs.png)                    |
+|        Left Trigger Button       |     Rift     |          |  U+E0E7 |                     ![](./images/rift-lt.png)                    |
+|       Right Trigger Button       |     Rift     |          |  U+E0E8 |                     ![](./images/rift-rt.png)                    |
+|             X Button             |     Rift     |          |  U+E0E9 |                     ![](./images/rift-x.png)                     |
+|             Y Button             |     Rift     |          |  U+E0EA |                     ![](./images/rift-y.png)                     |
```
Comment (thebigsmileXD) [inline README.md:105]: Bit big of a change, eh? Looks like alot useless additional spaces all of the sudden

Comment (HBIDamian) [reply README.md:105]: I didn't want to disturb the flow of how nice it looks in code view. :D 
image

Comment (thebigsmileXD) [reply README.md:105]: image

Comment (HBIDamian) [reply README.md:105]: Yeh, it was like that when I found it. I've just added 18 spaces to each side anyway. With the text editor I use, I can click multiple lines, so I added a ton of  extra spaces to make the rest fit in. 

Original (before I edited it): 
image

Comment (thebigsmileXD) [reply README.md:105]: Before
image
After
image
Added visual effect on the outcome: none
Added loadtime: 53ms to 93ms -> 40ms
Added spaces: 1504
Added noise in git: 94 lines

And why?

Obviousely trolling here, have a good night.

**Pull request merged**